mobile/src/libs/components/app/app.tsx

```diff
@@ -6,23 +6,27 @@ import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import SplashScreen from 'react-native-splash-screen';
 import { Provider as StoreProvider } from 'react-redux';
 
-import { Toast } from '#libs/components/components';
+import { Loader, Toast } from '#libs/components/components';
 import { useEffect } from '#libs/hooks/hooks';
 import { store } from '#libs/packages/store/store';
 import { Root as RootNavigation } from '#navigations/navigations';
 
+import { DELAY } from './libs/constants';
 import { styles } from './styles';
 
 const App: React.FC = () => {
   useEffect(() => {
-    SplashScreen.hide();
+    setTimeout(() => {
+      SplashScreen.hide();
+    }, DELAY);
   }, []);
 
   return (
     <StoreProvider store={store.instance}>
       <GestureHandlerRootView style={styles.root}>
         <NavigationContainer>
           <RootNavigation />
+          <Loader />
```

Comment:
```suggestion
```

Reply:
But where should I move this component then? If not here, then I'll have to add it to every component🤔

Reply:
if you pass isVisible here, then it's ok